pagopa/io-app
# chore(IT Wallet): [SIW-1012] Add playground screen for IT-Wallet flows

## Short description
This PR adds playground screen to test it wallet flows.

https://github.com/pagopa/io-app/assets/49342144/3843fbe9-f647-4e4d-8408-bd3b644d5f8a

## List of changes proposed in this pull request
- Update Profile Navigator
- Update Profile screen
- Add ItwPlayground screen

## How to test
Using `io-dev-api-server ` enable it-wallet toggle and tap on IT Wallet in playground section. A screen with some ListItemNav should appear.
